Examples/Python/src/Detector.cpp

```diff
@@ -34,6 +34,11 @@ void addDetector(Context& ctx) {
         .def("name", &IContextDecorator::name);
   }
 
+  {
+    py::class_<IBaseDetector, std::shared_ptr<IBaseDetector>>(mex,
```

Comment:
Do we actually need a binding for IBaseDetector? Since python is a dynamically typed language, I would have believed that we are fine without with, but maybe I'm wrong.